src/api/protectedApiClient.ts

```diff
@@ -165,6 +228,123 @@ const changePrivilegeLevel = (request: {
     .catch((e) => e);
 };
 
+const createTeam = (request: {
+  name: string;
+  bio: string;
+  inviteEmails: string[];
+}): Promise<void> => {
+  return AppAxiosInstance.post(ProtectedApiClientRoutes.CREATE_TEAM, request)
+    .then((r) => r.data)
```

Comment:
Personally, I think it's better to use `res` and `err`, as it's more readable/grep-able.